modules/bank.js

```diff
@@ -282,7 +282,7 @@ async function bankDiscount(interaction) {
       const withdraw = await Module.db.bank.addCurrency(withdrawal);
 
       interaction.editReply("Watch your DMs for the code you just redeemed!");
-      interaction.user.send(`You have redeemed ${gb}${withdraw.value} for a $${discount.amount} discount code in the LDS Gamers Store! <http://ldsgamers.com/shop>\n\nUse code __**${discount.code}**__ at checkout to apply the discount. This code will be good for ${discount.maxNumberOfUsages} use. (Note that means that if you redeem a code and don't use its full value, the remaining value is lost.)\n\nYou now have ${gb}${balance.balance + withdraw.value}.`);
+      interaction.user.send(`You have redeemed ${gb}${withdraw.value} for a $${discount.amount} discount code in the LDS Gamers Store! <http://ldsgamers.com/shop>\n\nUse code __**${discount.code}**__ at checkout to apply the discount. This code will be good for ${discount.maxNumberOfUsages} use. (Note that means that if you redeem a code and don't use its full value, the remaining value is lost.)\n\nYou now have ${gb}${balance.balance + withdraw.value}.`).catch(u.noop);
```

Comment:
If we're going to noop this one, let's modify the language in the interaction reply so they know who to reach out to if the DM doesn't make it to them.